apps/ideas/templates/advocate_europe_ideas/idea_detail.html

```diff
@@ -61,7 +64,7 @@
         </div>
         <div class="container-medium ideadetail-comments">
             <h4>{% trans "Write comment" %}</h4>
-            {% react_comments ideasketch %}
+            {% react_comments idea %}
```

Comment:
it should be the base idea, but is think if we wanted to add comments to the proposal we could just do {% react_comments idea.proposal %}